ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/service/internal/RemoteFileServiceImpl.java

```diff
@@ -28,8 +28,10 @@
 import eu.elixir.ega.ebi.commons.shared.dto.MyExternalConfig;
 import eu.elixir.ega.ebi.commons.shared.service.DownloaderLogService;
 import eu.elixir.ega.ebi.commons.shared.service.FileInfoService;
-import eu.elixir.ega.ebi.dataedge.config.*;
 import eu.elixir.ega.ebi.dataedge.dto.*;
+import eu.elixir.ega.ebi.dataedge.exception.InternalErrorException;
+import eu.elixir.ega.ebi.dataedge.exception.NoContentException;
```

Comment:
@anandmohan777 Can this be handled with the shared exception in commons 
`NotFound | 404 | The resource requested was not found` ? I think the concept it's exactly the same and we don't need duplicities :)

Reply:
NoContentException(204) is different from NotFoundException(404). NotFoundException can't be used here.

Reply:
Agree :-) Let's move then NoContentException to commons then